core/src/main/scala/caliban/execution/Executor.scala

```diff
@@ -111,7 +112,13 @@ object Executor {
           i += 1
           remaining = remaining.tail
         }
-        reduceList(lb.result(), Types.listOf(currentField.fieldType).fold(false)(_.isNullable))
+        reduceList(
```

Comment:
Is this actually faster? `fold` is `inline`.

Reply:
Inlining via the `@inline` annotation only works in Scala 2 and only if it's enabled via the `-opt-inline-from:` compiler flag (which we don't have enabled - I think we might want to enable that one).

Reply:
ah :/